draft-welzl-ccwg-ratelimited-increase.md

```diff
@@ -185,30 +188,37 @@ The specification and the ns-2 and ns-3 implementations are in conflict with rul
 
 {{Section 5.8 of !RFC9438}} says:
 
->Cubic doesn't increase cwnd when it's limited by the sending application or rwnd.
+>Cubic does not increase cwnd when it is limited by the sending application or rwnd.
```

Comment:
```suggestion
>Cubic doesn't increase cwnd when it's limited by the sending application or rwnd.
```

I suggest to undo this because it's a literal quote. This is really how it's written in the RFC.

Reply:
Maybe put it under quotation marks?